src/components/Integrations/Shopify/Shopify.css

```diff
@@ -0,0 +1,8 @@
+img {
+    width: 10%;
```

Comment:
10% regarding what?

Reply:
The outer parent container.
![image](https://user-images.githubusercontent.com/2439363/60973821-56e94880-a2ff-11e9-99e3-448662164266.png) mmm doesn't seem to be a good idea. I will change it.